packages/office-ui-fabric-react/src/components/Stack/StackItem/StackItem.styles.ts

```diff
@@ -10,7 +10,13 @@ const alignMap: { [key: string]: string } = {
   end: 'flex-end'
 };
 
-export const styles: IStackItemComponent['styles'] = (props, theme): IStackItemStylesReturnType => {
+const baseTokens: IStackItemComponent['tokens'] = {
+  margin: 0
```

Comment:
Hmm, dumb question. Is it better to define this as 0 or not define it by default? Is there a difference? (This is definitely affecting default styling output...)

Looking further at the PR, I think it would be better to leave this undefined as that seems more consistent with our styling approach in general. (Unless there's a reason it needs to be defined as 0 by default.)